nextroute/model_sanity_check.go

```diff
@@ -1,11 +1,21 @@
 package nextroute
 
 import (
+	"context"
 	"time"
 
 	"github.com/nextmv-io/sdk/connect"
+	"github.com/nextmv-io/sdk/run/schema"
 )
 
+// SanityCheckOptions are the options for the sanity check.
+type SanityCheckOptions struct {
+	Enable bool `json:"enable" usage:"enable the sanity check" default:"false"`
+	// Duration is the maximum duration allowed for the sanity check to run.
```

Comment:
I think the comment can go away, as we already have a `usage` field

Reply:
It is gone